english/hulk/stop-hulk.md

```diff
@@ -0,0 +1 @@
+You rocket into the sky and head straight for Gotham. You arrive to find half of the city completely destroyed. You are just in time to watch the Hulk kill Batman. 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.